store/cluster_shard.go

```diff
@@ -33,11 +33,18 @@ import (
 	"github.com/apache/kvrocks-controller/consts"
 )
 
+const (
+	NotMigratingString = "not-migrating"
```

Comment:
We could keep the field null if it's not migrating.

Reply:
Hmm to have the field null, il have to make MigrationSlot field a pointer again.

Which I'm ok with doing. Il make the changes and see if you are ok with how it looks